Update newsletter sponsorship pricing (#2162)
writethedocs/www: docs/sponsorship/newsletter.rst

Our audience is made up of technical writers and decision makers in the software documentation industry.
The current newsletter stats are:

* Over {{ newsletter_subs }} subscribers

> * 42% open rate

Reviewer: Is it worth highlight just how high this number is compared to industry average? @goatlady

Reviewer 2: The problem is that open rates are kind of dead as a metric since Apple started blocking tracking pixels by default and Google started caching images from bulk senders a couple of years ago. Your open rate is almost certainly much higher than that, but because most of the content is in the email itself rather than outbound links, MailChimp can't count the click-throughs as confirmed opens. 

Having said that, regardless of industry, an open rate of over 30% is considered excellent. There's an industry breakdown from 2022 data here: https://www.campaignmonitor.com/resources/knowledge-base/what-are-good-email-metrics/

> The best idea of who is visiting our site is the information we have from our conferences.

Reviewer: ```suggestion
We don't exact demographic data for our newsletter subscribers, but according to the data gathered from our conferences, our audience is made up of:
```

> Our audience is made up of:

Reviewer: ```suggestion

```

.. include:: /include/demographics.rst

Sponsored entry format
----------------------

> Our sponsorship format is similar to our normal newsletter entries.

Reviewer: ```suggestion
Our sponsorship format is similar to our normal newsletter entries, with an added company logo.
```
